test/integration/framework/apiserver.go

```diff
@@ -62,26 +60,28 @@ func RunControlPlane(t *testing.T) (*rest.Config, StopFunc) {
 	}
 	patchCRDConversion(crds, webhookOpts.URL, webhookOpts.CAPEM)
 
-	env.CRDs = crdsToRuntimeObjects(crds)
+	runtimeCrds := crdsToRuntimeObjects(crds)
 
-	config, err := env.Start()
-	if err != nil {
-		t.Fatalf("failed to start control plane: %v", err)
-	}
+	envtest.CreateCRDs(config, runtimeCrds)
+
+	envtest.WaitForCRDs(config, runtimeCrds, envtest.CRDInstallOptions{
+		PollInterval: 100 * time.Millisecond,
+		MaxTime:      4 * time.Second,
+	})
```

Comment:
use the `envtest.CreateCRDs` and `envtest.WaitForCRDs` to install the cert-manager CRDs and "WaitForCRDs waits for the CRDs to appear in discovery.".
inspired by: https://github.com/kubernetes-sigs/controller-runtime/blob/3c54acbad091d40ef80d467943c5d7126c4b8291/pkg/envtest/crd.go#L97-L113

Reply:
These both return errors which we should check.